modules/integration/tests-integration/pom.xml

```diff
@@ -109,5 +109,9 @@
             <artifactId>org.wso2.carbon.um.ws.api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.template.mgt</artifactId>
+        </dependency>
     </dependencies>
 </project>
```

Comment:
Add a blank line

Reply:
fixed with https://github.com/wso2/product-is/pull/9220.